Bottom: 95460fe5374d44c039db936540a7cc68bbf1716c
Top:    0f2165b2dfdbc6f63ebc5485c118b7b9bf464ac9
Author: Edward "Teddy" Brow <edward.teddy.brow@gmail.com>
Date:   2017-04-21 16:46:45 -0700

update .gitignore - don't ignore Makefiles!


---

diff --git a/.gitignore b/.gitignore
index 2920710..38b4ff7 100644
--- a/.gitignore
+++ b/.gitignore
@@ -1,6 +1,7 @@
 *
 !*/
 !*.*
+!*Makefile*
 
 *.o
 *.swp
diff --git a/01_12/Makefile b/01_12/Makefile
new file mode 100644
index 0000000..fd9f01a
--- /dev/null
+++ b/01_12/Makefile
@@ -0,0 +1,5 @@
+main:
+	g++ -o main main.cc
+
+clean:
+	rm main
